src/main/kotlin/com/csbroker/apiserver/model/ShortProblem.kt

```diff
@@ -84,7 +84,17 @@ class ShortProblem(
             scoreList.count { it == this.score },
             scoreList.count { it != this.score },
             totalSolved,
-            this.answer.length
+            this.answer.length,
+            this.isEnglish()
         )
     }
+
+    private fun isEnglish(): Boolean {
+        for (c in this.answer) {
+            if (c !in 'A'..'Z' && c !in 'a'..'z' && c !in '0'..'9') {
+                return false
+            }
+        }
+        return true
+    }
```

Comment:
코드를 조금 더 간결하게 해보는게 어떨까요!
```suggestion
    private fun isEnglish(): Boolean {
         return this.answer.all { it.isLetter() }
    }
```

Reply:
@ekzm8523 isLetter 가 영문자인지 판단하는게 아니라서, 한글이여도 true가 반환됩니다.